content/openapi/introduction.md

```diff
@@ -51,6 +56,7 @@ Once we create a document, we can call the `setup()` method. It accepts:
 1. the path to mount the Swagger UI
 2. an application instance
 3. the document object instantiated above
+4. optional configuration parameter
```

Comment:
Should we mention what kind of configuration parameters can go here?

Reply:
I mentioned it in the new commit specifically. Thanks for the suggestion.